src/core/rtkQuery/splitV3Api/seriesApi.ts

```diff
@@ -200,6 +200,15 @@ const seriesApi = splitV3Api.injectEndpoints({
       }),
     }),
 
+    changeSeriesImage: build.mutation<ResponseType, { seriesId: string, imageType: string, params: object }>({
+      query: ({ imageType, params, seriesId }) => ({
+        url: `Series/${seriesId}/Images/${imageType}`,
+        method: 'PUT',
+        body: params,
+      }),
+      invalidatesTags: ['SeriesAniDB'],
+    }),
```

Comment:
We need a new tag for `'SeriesImages'` to invalidate, or to just reuse the existing `'SeriesAniAD'` tag for the image endpoint so we can refresh the image data.

Reply:
I added this so the series poster would change when a new default is selected since the series poster already is using ``SeriesAniDB``, but I can change it.

Reply:
Why do we need a new tag? We don't use separate image tag anywhere as far as I can tell.

Reply:
The tag would need to be added. It's not currently used since we don't have any mutations that modify the image data. it's also neither hooked up to update when the series data is updated.

Reply:
I added a new one for images and use that one now.

Reply:
But that tag `SeriesImages` makes no sense, for example you have  images fetched for multiple series in cache, then by invalidating this tag you will be resetting all the other series images as well because your `SeriesImages` tag is not specific enough, it does not have series id in it. See rtkq docs here; https://redux-toolkit.js.org/rtk-query/usage/automated-refetching#invalidating-tags.

`SeriesAniDB` tag it is also potentially has the issue.